tools/stonith_admin.c

```diff
@@ -605,9 +604,18 @@ main(int argc, char **argv)
 
   done:
     g_strfreev(processed_args);
-    g_clear_error(&error);
     pcmk__free_arg_context(context);
 
+    if (error != NULL) {
+        if (out != NULL) {
+            out->err(out, "%s: %s", g_get_prgname(), error->message);
+        } else {
+            fprintf(stderr, "%s: %s\n", g_get_prgname(), error->message);
+        }
+
+        g_clear_error(&error);
+    }
```

Comment:
This seems used often enough to merit functionizing and using in all tools

Reply:
Yeah, great idea.  This is exactly the sort of think I am looking to do in this PR.